lib/topics/state_and_behaviour.rb

```diff
@@ -2,4 +2,27 @@
 
 # exercise state and behaviour
 module StateAndBehaviour
+  # implement class Car
+  class Car
+    attr_accessor :year, :color, :model, :current_speed
+
+    def self.default_car
+      new
+    end
+
+    def initialize(params)
+      @year           = params.fetch(:year, '2000')
```

Comment:
`fetch` is good! But it has the behaviour like this: `{a: nil}.fetch(:a, 1) => nil`. That means `Car.new(year: nil)` will save `nil` to `@year` variable. If that behaviour is allowed by design, I would suggest using [keyword arguments](https://robots.thoughtbot.com/ruby-2-keyword-arguments) here